app/controllers/trip_families_controller.rb

```diff
@@ -0,0 +1,41 @@
+class TripFamiliesController < ApplicationController
+  before_action :load_trip
+
+  def index
+    @trip_families = @trip.trip_families.all
+  end
+
+  def show
+    @trip_family = @trip.trip_families.find(params[:id])
+
+    respond_with @trip_family, location: -> { trip_family_path }
+  end
+
+  def new
+    @trip_family = @trip.trip_families.new
+  end
+
+  def create
+    @trip_family = @trip.trip_families.create(trip_family_params)
+
+    respond_with @trip, @trip_family,
+      location: -> { trip_path([@trip]) }
+  end
+
+  private
+
+  def trip_family_params
+    params.require(:trip_family).permit(
+      :total_girls,
+      :total_boys,
+      :total_techers,
+      :total_bus_drivers,
+      :family_id,
+      :trip_id,
+    )
+  end
+
+  def load_trip
```

Comment:
If you rename this method to `trip` you can just call `trip` whenever you need it.
And also will not need to call the `before_action`

Reply:
If you define this method as a `helper_method`, you will be able to use it in the views.

Reply:
Done in sha `751fbba`